### DISCUSSION TITLE
How to prevent user to minimize Textarea field?

### DISCUSSION THREAD

**Discussion by hiimnhan** [cloudscape-design/components#768] [Q&A] [answered]
In the Textarea component, I set default rows to 5 and I don't want user to minimize the Textarea field. How can I do that?

**Comment by Al-Dani** [comment, marked as answer]
Hey [hiimnhan](https://github.com/hiimnhan),
Sorry for the late response!

Currently it's not possible to prevent Textarea component from resizing. 

Could you share more details about your use-case? Will the text be editable?

Best,
Aleksandra
